src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByAuthorizationCodeParameterBuilder.cs

```diff
@@ -114,13 +114,9 @@ public AcquireTokenByAuthorizationCodeParameterBuilder WithPkceCodeVerifier(stri
         /// <returns>The builder to chain the .With methods</returns>
         public AcquireTokenByAuthorizationCodeParameterBuilder WithCcsRoutingHint(string userObjectIdentifier, string tenantIdentifier)
         {
-            if (string.IsNullOrEmpty(userObjectIdentifier))
+            if (string.IsNullOrEmpty(userObjectIdentifier) || string.IsNullOrEmpty(tenantIdentifier))
```

Comment:
I guess CCS work with ADFS and B2C?

Reply:
ADFS uses it, B2C will ignore it, but there is another issue for [B2C](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2748)